### PR TITLE
Major/update footer component

### PR DESCRIPTION
**Pull request by eduardojmatos**
**CHANGELOG** :memo:

* Our footer is changing and we need to remove old CSS of the old footer.
* This PR moves some CSS of our site to reuse on another system (aka Tanya);
* I renamed `footer__list--service` to `footer__list--on-column`, because `service` is a internal nomenclature of a data model.

### CODE REVIEW

**Comment by eduardojmatos** [comment]
[[Tanya] Footer na página de Preços](https://trello.com/c/vvqlo2At/878-tanya-footer-na-p%C3%A1gina-de-pre%C3%A7os)

File: docs/demo/gaiden-css/components/footer/demo.html [hunk 1]
```diff
@@ -12,65 +12,79 @@
...
-        </div>
-      <div class="footer__company-disclaimer" role="presentation">
-        <p>©2011-2017, GetNinjas Serviços de Internet. CNPJ 14.127.813/0001-51</p> - <address >Av. Rebouças, 2472 - Pinheiros São Paulo - SP</address>
+  <footer class="footer">
+    <nav class="footer__wrapper container" role="menubar" aria-label="Menu rodapé">
+      <div class="footer__block col-small-12 col-medium-6 col-normal-6 col-large-4">
+        <span class="footer__title">
+          Serviços
+        </span>
+        <ul class="footer__list footer__list--on-column col-small-6 col-medium-6 col-normal-6  col-large-6" role="menu" aria-label="Menu rodapé - serviços">
```
Comment (williamcosta) [inline docs/demo/gaiden-css/components/footer/demo.html]: Não precisamos colocar todos os `col-6` já que ele é o mesmo tamanho desde o small.

File: docs/demo/gaiden-css/components/footer/demo.html [hunk 2]
```diff
@@ -12,65 +12,79 @@
...
+          Institucional
+        </span>
+        <ul class="footer__list" role="menu" aria-label="Menu rodapé - institucional">
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Quem Somos" href="https://www.getninjas.com.br/quem-somos">Quem Somos</a></li>
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Trabalhe Conosco" href="https://jobs.kenoby.com/getninjas" target="_blank">Trabalhe Conosco</a></li>
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Profissionais Verificados" href="http://comunicacao.getninjas.com.br/selo-ninja-verificado">Profissionais Verificados</a></li>
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Guia do GetNinjas" href="https://www.getninjas.com.br/guia" rel="noopener noreferrer">Guia do GetNinjas</a></li>
+        </ul>
       </div>
+      <div class="footer__block col-small-12 col-medium-6 col-normal-6 col-large-4">
```
Comment (williamcosta) [inline docs/demo/gaiden-css/components/footer/demo.html]: Aqui não precisariamos do `col-normal-6`.

File: src/scss/components/_footer.scss [hunk 3]
```diff
@@ -6,157 +6,175 @@
...
 
-  &__company-disclaimer {
-    @include media(medium) {
-      text-align: inherit;
+  &__block {
+    @include media(small) {
+      align-content: flex-start;
+      margin-top: get-space(box, large);
     }
+    align-content: flex-start;
```
Comment (williamcosta) [inline src/scss/components/_footer.scss]: Essas propriedades não teria que estar antes do `breakpoint`?

File: src/scss/gaiden.scss [hunk 4]
```diff
@@ -12,9 +11,10 @@
 @import 'tools/mixins/map_deep_get';
 @import 'tools/mixins/generate_button_sizes';
 @import 'tools/mixins/generate_button_types';
+@import 'tools/mixins/get_border';
 @import 'tools/mixins/get_color';
 @import 'tools/mixins/get_shadow';
-@import 'tools/mixins/get_border';
+@import 'tools/mixins/get-space';
```
Comment (williamcosta) [inline src/scss/gaiden.scss]: `get-space` não devaria estar com **underscore** para seguir o padrão que estamos usando?

File: src/scss/gaiden.scss [hunk 5]
```diff
@@ -12,9 +11,10 @@
...
 @import 'tools/mixins/get-weight';
```
Comment (williamcosta) [inline src/scss/gaiden.scss]: @eduardojmatos se puder já aproveitar e mudar aqui para **underscore**.

File: docs/demo/gaiden-css/components/footer/demo.html [hunk 6]
```diff
@@ -12,65 +12,79 @@
...
+          <a href="#"><img alt="GetNinjas" class="footer__image" src="/demo/gaiden-css/images/svg/getninjas.svg"></a>
+        </li>
+        <ul class="footer__institutional-list space-element-large">
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Imprensa" href="https://www.getninjas.com.br/imprensa">Imprensa</a></li>
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Ajuda" href="https://getninjas.desk.com/" target="_blank">Ajuda</a></li>
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Termos de uso" href="https://www.getninjas.com.br/termos-de-uso">Termos de uso</a></li>
+        </ul>
+      </ul>
+    </div>
+    <address role="conentinfo" class="footer__copyright">©2011-2018, GetNinjas Serviços de Internet. CNPJ 14.127.813/0001-51 - Av. Rebouças, 2472 - Pinheiros São Paulo - SP</address>
```
Comment (iondrimba) [inline docs/demo/gaiden-css/components/footer/demo.html]: @eduardojmatos `conentinfo`

File: src/scss/components/_footer.scss [hunk 7]
```diff
@@ -6,157 +6,175 @@
...
-    background-color: get-color(blackNeutral);
-    padding: {
-      bottom: 24px;
-      top: 24px;
-    };
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: 900;
```
Comment (iondrimba) [inline src/scss/components/_footer.scss]: @eduardojmatos Vale usar o helper de font-weight? `get-weight`

File: src/scss/components/_footer.scss [hunk 8]
```diff
@@ -6,157 +6,175 @@
...
+          &:first-child {
+            margin: auto;
+          }
+        }
+      }
+
+      color: get-color(punch);
+      display: flex;
+      font-size: 12px;
+      font-weight: bold;
```
Comment (iondrimba) [inline src/scss/components/_footer.scss]: @eduardojmatos Vale usar o helper de font-weight? `get-weight`

File: docs/demo/gaiden-css/components/footer/demo.html (hunk 6 above)
Comment (eduardojmatos) [reply docs/demo/gaiden-css/components/footer/demo.html]: @iondrimba nice. Our home is with this error...

File: src/scss/components/_footer.scss (hunk 7 above)
Comment (eduardojmatos) [reply src/scss/components/_footer.scss]: Yes. Scorpion is with this font-weight without helper...

**Comment by iondrimba** [comment]
@eduardojmatos Some small points:

1. Can we add a logo or some image to avoid the broken image i.e

captura de tela 2018-01-09 as 11 00 23 am

2. Somehow the group of links are not inlined running the demo locally
3. I noticed the naming was changed just a little bit from `footer__institutional--list`  to `footer__institutional-list` (credits @williamcosta)

File: src/scss/components/_footer.scss (hunk 3 above)
Comment (eduardojmatos) [reply src/scss/components/_footer.scss]: @williamcosta our sass lint required mixins before any declaration. Let's discuss this on our #frontend channel :)

File: docs/demo/gaiden-css/components/footer/demo.html (hunk 1 above)
Comment (eduardojmatos) [reply docs/demo/gaiden-css/components/footer/demo.html]: Yes. We need to fix this on our home Sass, on Scorpion project

File: src/scss/gaiden.scss (hunk 4 above)
Comment (eduardojmatos) [reply src/scss/gaiden.scss]: @williamcosta yes. I'll fix this now. `get-weight` has the same problem, but we don't saw in older PR.

**Comment by eduardojmatos** [comment]
@iondrimba nice!

1. Yes
2. Exactly... I was trying to reuse `footer__institutional`, but dont makes sense. Fixing now...
3. This is not what I see on our code-base, at Scorpion. Is this change is on some PR?

**Comment by eduardojmatos** [comment]
@iondrimba about point 3, I'm moving `footer__institutional--list` to a `footer__institutional-list`, because this seletor was not modifying the `footer__institutional`. His behaviour is to be a new element of `footer__institutional`.

**Comment by iondrimba** [comment]
@eduardojmatos Nice! 👍

**Pull request merged**